[2021-11-17 01:06:34,473] {processor.py:163} INFO - Started process (PID=44827) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:06:34,474] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 01:06:34,474] {logging_mixin.py:109} INFO - [2021-11-17 01:06:34,474] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:06:34,489] {logging_mixin.py:109} INFO - [2021-11-17 01:06:34,487] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 01:06:34,505] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:06:34,508] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.034 seconds
[2021-11-17 01:07:42,161] {processor.py:163} INFO - Started process (PID=44867) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:07:42,161] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 01:07:42,162] {logging_mixin.py:109} INFO - [2021-11-17 01:07:42,162] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:07:42,177] {logging_mixin.py:109} INFO - [2021-11-17 01:07:42,175] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 01:07:42,192] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:07:42,196] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.035 seconds
[2021-11-17 01:08:56,002] {processor.py:163} INFO - Started process (PID=44909) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:08:56,004] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 01:08:56,004] {logging_mixin.py:109} INFO - [2021-11-17 01:08:56,004] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:08:56,020] {logging_mixin.py:109} INFO - [2021-11-17 01:08:56,019] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 01:08:56,035] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:08:56,039] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.037 seconds
[2021-11-17 01:10:09,672] {processor.py:163} INFO - Started process (PID=44974) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:10:09,673] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 01:10:09,673] {logging_mixin.py:109} INFO - [2021-11-17 01:10:09,673] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:10:09,689] {logging_mixin.py:109} INFO - [2021-11-17 01:10:09,688] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 01:10:09,708] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:10:09,712] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.041 seconds
[2021-11-17 01:11:23,355] {processor.py:163} INFO - Started process (PID=45014) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:11:23,356] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 01:11:23,356] {logging_mixin.py:109} INFO - [2021-11-17 01:11:23,356] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:11:23,371] {logging_mixin.py:109} INFO - [2021-11-17 01:11:23,370] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 01:11:23,387] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:11:23,391] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.036 seconds
[2021-11-17 01:12:37,204] {processor.py:163} INFO - Started process (PID=45052) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:12:37,205] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 01:12:37,205] {logging_mixin.py:109} INFO - [2021-11-17 01:12:37,205] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:12:37,221] {logging_mixin.py:109} INFO - [2021-11-17 01:12:37,219] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 01:12:37,236] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:12:37,240] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.036 seconds
[2021-11-17 01:13:50,691] {processor.py:163} INFO - Started process (PID=45088) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:13:50,691] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 01:13:50,691] {logging_mixin.py:109} INFO - [2021-11-17 01:13:50,691] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:13:50,706] {logging_mixin.py:109} INFO - [2021-11-17 01:13:50,705] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 01:13:50,721] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:13:50,725] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.034 seconds
[2021-11-17 01:15:04,510] {processor.py:163} INFO - Started process (PID=45124) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:15:04,511] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 01:15:04,511] {logging_mixin.py:109} INFO - [2021-11-17 01:15:04,511] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:15:04,525] {logging_mixin.py:109} INFO - [2021-11-17 01:15:04,524] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 01:15:04,541] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:15:04,545] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.035 seconds
[2021-11-17 01:20:30,545] {processor.py:163} INFO - Started process (PID=45202) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:20:30,546] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 01:20:30,546] {logging_mixin.py:109} INFO - [2021-11-17 01:20:30,546] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:20:30,560] {logging_mixin.py:109} INFO - [2021-11-17 01:20:30,558] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 01:20:30,576] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 01:20:30,578] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.034 seconds
[2021-11-17 08:50:38,035] {processor.py:163} INFO - Started process (PID=45393) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 08:50:38,035] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 08:50:38,035] {logging_mixin.py:109} INFO - [2021-11-17 08:50:38,035] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 08:50:38,049] {logging_mixin.py:109} INFO - [2021-11-17 08:50:38,047] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 08:50:38,064] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 08:50:38,067] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.033 seconds
[2021-11-17 13:30:56,540] {processor.py:163} INFO - Started process (PID=45491) to work on /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 13:30:56,541] {processor.py:625} INFO - Processing file /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py for tasks to queue
[2021-11-17 13:30:56,541] {logging_mixin.py:109} INFO - [2021-11-17 13:30:56,541] {dagbag.py:500} INFO - Filling up the DagBag from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 13:30:56,559] {logging_mixin.py:109} INFO - [2021-11-17 13:30:56,557] {dagbag.py:334} ERROR - Failed to import: /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
Traceback (most recent call last):
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py", line 84, in <module>
    virtualenv_task = PythonVirtualenvOperator(
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 342, in __init__
    raise AirflowException('PythonVirtualenvOperator requires virtualenv, please install it.')
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2021-11-17 13:30:56,574] {processor.py:639} WARNING - No viable dags retrieved from /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py
[2021-11-17 13:30:56,577] {processor.py:171} INFO - Processing /Users/sravanisigadam/PycharmProjects/Airflow/venv/lib/python3.9/site-packages/airflow/example_dags/example_python_operator.py took 0.038 seconds
